en/includes/guides/users/attributes/manage-attributes.md

```diff
@@ -207,6 +207,40 @@ To delete an attribute:
 
     Only custom attributes can be deleted.
 
+## Manage Identity Attributes
+
+By default, identity claim values are stored in the JDBC datasource configured in the `deployment.toml` file. If required, you can configure WSO2 IS to store the claim values in another user store as well.
```

Comment:
Is this doc for IS only or for both IS and asgardeo? if this is for both, we may need to use placeholders for the product name

Reply:
Only applicable for IS. however, display content only for IS condition needed to be added. d3807d00f7a43810a30ecaedba821d4b0d5b763c